src/content/reference/manifest.mdx

```diff
@@ -577,7 +591,17 @@ dev:
       - .:/app
     services:
       - name: worker
-        command: ["celery", "worker", "-A", "myproject.celeryconf", "-Q", "default", "-n", "default@%h"]
+        command:
```

Comment:
I prefer the old formatting because it was more compact. Since it doesn't add value to understanding how to use `okteto`, I prefer having it as small as possible to not draw attention awat

Reply:
this is not for the PR, it got formatted and i commited it, i will remove this change